tests/logfile.json

```diff
@@ -1,8 +1 @@
[36mconst\u001b[39m thisComponent \u001b[33m=\u001b[39m shallow(\u001b[33m<\u001b[39m\u001b[33mOrderSuccessful\u001b[39m \u001b[33m/\u001b[39m\u001b[33m>\u001b[39m)\u001b[33m;\u001b[39m\u001b[0m\n    \u001b[0m \u001b[90m 6 |\u001b[39m\u001b[0m\n\n      \u001b[2mat Resolver.resolveModule (\u001b[22mnode_modules/jest-runtime/node_modules/jest-resolve/build/index.js\u001b[2m:306:11)\u001b[22m\n      \u001b[2mat Object.<anonymous> (\u001b[22m\u001b[0m\u001b[36msrc/__tests__/OrderSuccessful.test.js\u001b[39m\u001b[0m\u001b[2m:3:1)\u001b[22m\n","name":"/Users/oluwatimileyinakinpelu/Documents/Dufuna-CodeCamp-21/submissions/oluwatimileyinAkinpelu/react/react-app/src/__tests__/OrderSuccessful.test.js","startTime":1658890991056,"status":"failed","summary":""},{"assertionResults":[],"coverage":{},"endTime":1658890991056,"message":"  \u001b[1m● \u001b[22mTest suite failed to run\n\n    Cannot find module 'react' from '../../../../../../node_modules/react-router-dom/umd/react-router-dom.development.js'\n\n    Require stack:\n      /Users/oluwatimileyinakinpelu/node_modules/react-router-dom/umd/react-router-dom.development.js\n      /Users/oluwatimileyinakinpelu/node_modules/react-router-dom/main.js\n      src/App.js\n      src/__tests__/App.test.js\n\n      \u001b[2mat Resolver.resolveModule (\u001b[22mnode_modules/jest-runtime/node_modules/jest-resolve/build/index.js\u001b[2m:306:11)\u001b[22m\n      \u001b[2mat \u001b[22m../../../../../../node_modules/react-router-dom/umd/react-router-dom.development.js\u001b[2m:12:83\u001b[22m\n      \u001b[2mat Object.<anonymous> (\u001b[22m../../../../../../node_modules/react-router-dom/umd/react-router-dom.development.js\u001b[2m:15:2)\u001b[22m\n","name":"/Users/oluwatimileyinakinpelu/Documents/Dufuna-CodeCamp-21/submissions/oluwatimileyinAkinpelu/react/react-app/src/__tests__/App.test.js","startTime":1658890991056,"status":"failed","summary":""},{"assertionResults":[{"ancestorTitles":[],"failureMessages":[],"fullName":"renders without crashing","location":null,"status":"passed","title":"renders without crashing"},{"ancestorTitles":[],"failureMessages":[],"fullName":"test case: check header title","location":null,"status":"passed","title":"test case: check header title"},{"ancestorTitles":[],"failureMessages":[],"fullName":"test case: check intial counter value","location":null,"status":"passed","title":"test case: check intial counter value"},{"ancestorTitles":[],"failureMessages":[],"fullName":"test case: expected images","location":null,"status":"passed","title":"test case: expected images"},{"ancestorTitles":[],"failureMessages":[],"fullName":"test case: check submit button","location":null,"status":"passed","title":"test case: check submit button"}],"endTime":1658890991045,"message":"","name":"/Users/oluwatimileyinakinpelu/Documents/Dufuna-CodeCamp-21/submissions/oluwatimileyinAkinpelu/react/react-app/src/__tests__/CartPage.test.js","startTime":1658890989981,"status":"passed","summary":""}],"wasInterrupted":false}
```

Comment:
Well done @cddunni! Can you try running the test file again and then push. Just DM me peradventure you experience blockers so that we can resolve this together once and for all. Thanks!

Reply:
Alright